src/tests/tracing/eventpipe/simpleprovidervalidation/simpleprovidervalidation.cs

```diff
@@ -14,13 +14,6 @@
 
 namespace Tracing.Tests.SimpleProviderValidation
 {
-    public sealed class MyEventSource : EventSource
-    {
-        private MyEventSource() {}
-        public static MyEventSource Log = new MyEventSource();
-        public void MyEvent() { WriteEvent(1, "MyEvent"); }
-    }
-
     public class ProviderValidation
```

Comment:
This test is no longer targeting custom event sources and is only for runtime events now? Is that because we're just relying on the `providervalidation` test to do that? Maybe this test should be renamed if it is just for runtime events.

Reply:
Changed test name to reflect the new usage